dev/ci/ci-basic-overlay.sh

```diff
@@ -79,8 +79,8 @@
 ########################################################################
 # VST
 ########################################################################
-: ${VST_CI_BRANCH:=less_init_plugins}
-: ${VST_CI_GITURL:=https://github.com/letouzey/VST.git}
+: ${VST_CI_BRANCH:=master}
+: ${VST_CI_GITURL:=https://github.com/Zimmi48/VST.git}
```

Comment:
That would do the trick but I'm not sure we want that as this could reduce the testability of some PRs... (who add things to `Compat86.v`).
BTW I must insist that this issue is not specific to this development so the fix should be considered independently.